examples/workflows/run_ensemble_model_noise.py

```diff
@@ -28,21 +28,28 @@
 def generate_model_noise_correlated(
     x,
     time_step,
-    reddening,
     device,
+    reddening,
     noise_injection_amplitude,
+    normalize=False,
+    center=0,
+    scale=1,
 ):
     shape = x.shape
     dt = torch.tensor(time_step.total_seconds()) / 3600.0
     noise = noise_injection_amplitude * dt * brown_noise(shape, reddening).to(device)
-    return x * (1.0 + noise)
+    if normalize:
+        x += noise
+    else:
+        x += noise * scale
+    return x
```

Comment:
Just return the source instead of applying it.

```suggestion
    return x * noise
```